main/views.py

```diff
@@ -71,4 +73,25 @@ def post(self, request, *args, **kwargs):
 @login_required
 def log_out(request):
     logout(request)
-    return redirect('/home/')
\ No newline at end of file
+    return redirect('/home/')
+
+# The LoginRequiredMixin it's the same of login_required but for classes
+class VeiwList(LoginRequiredMixin,TemplateView):
```

Comment:
change the class name to SuggestedJobsView the name isn't clear.

Reply:
Done